.buildkite/pipeline.yml

```diff
@@ -58,7 +58,7 @@ steps:
       - "result/bin/coffer"
 
   - label: xrefcheck
-    command: nix run -f ci.nix xrefcheck -c xrefcheck --no-progress -m full --ignored tests/golden/helpers
+    command: nix run -f ci.nix xrefcheck -c xrefcheck --ignored tests/golden/helpers
```

Comment:
Should this be turned into a glob pattern? `tests/golden/helpers/**/*`

Reply:
Oh wait, I don't think we even need to ignore this anymore.

Previosly, we were using `--ignored` here to ignore markdown files contained inside the `bats` / `bats-assert` / `bats-support` submodules, e.g.:

* tests/golden/helpers/bats/README.md
* tests/golden/helpers/bats-support/README.md
* tests/golden/helpers/bats-assert/README.md

But, because these are submodules, those files are not really tracked in our repository. `git ls-files` only returns their directories, not the files inside:

```
$ git ls-files
...
tests/golden/helpers/bats
tests/golden/helpers/bats-assert
tests/golden/helpers/bats-support
...
```

And since 0.2.2 uses `git ls-files`, we don't need to use `--ignored` for this anymore.